### PR TITLE
PIP-1421-update-docs

### PR DESCRIPTION
**Pull request by paul-sud**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
```
Comment (ottojolanki) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: Templating these is a great idea. Definitely will adopt to my pipelines.

File: docs/reference.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+Runs the restriction sites file generation step only.
+
+*Required inputs*
+* `reference_fasta`
+* `restriction_site_locations_only`, should be set to `true`
+
+### Input descriptions
+
+* `fastq` is a twice nested array of input fastqs. The outermost level corresponds to the biological replicates. Each biological replicate then contains one or more technical replicates. Each technical replicate is an array containing two paired fastq files.
```
Comment (ottojolanki) [inline docs/reference.md]: I would add an example here. Nested arrays seem to confuse people.

**Review by ottojolanki — COMMENTED** [review]
Looks very good. I would consider adding an example of fastq input with couple of different biological/technical replicate scenarios.

File: docs/reference.md (hunk 2 above)
Comment (paul-sud) [reply docs/reference.md]: Yes it is surprisingly hard to express it in words. I will add an example.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (paul-sud) [reply .github/ISSUE_TEMPLATE/bug_report.md:1]: I stole the idea from ChIP/ATAC :)

**Pull request merged**